### ISSUE TITLE
FEATURE: Allow exclusions to be family based

### ISSUE THREAD

**Issue by nwehrman** [arcanis/secretsanta#33]
For multi family matching it takes a while to configure each individual person and who they can't be matched with. Allow family creation so you can set Name1, Name2, Name3 as part of family 1 and Name4,Name7,Name8,Name9 as part of family 2 and auto exclude from matching within the family. This could also apply to business units within company parties.

**Comment by arcanis** [comment]
I think it'd make sense - the tricky part will be finding out a UI that's both opt-in and not too confusing. I have some ideas, I'll see if I can get something done later this week. Worst case, this will be for next year 😄

**Comment by Rickdiculousme** [comment]
good idea! check out the Excel helper tool I made in the meantime if you want. I made it a few years ago and kept improving it so I figured I'd share if it helps others https://github.com/Rickdiculousme/SecretSanta-Excel-Helper

**Comment by Rickdiculousme** [comment]
This might help a bit for large input cases with exclusions. The output is still a bit manual. I made this Excel helper tool a few years ago and kept improving it so I figured I'd share if it helps others https://github.com/Rickdiculousme/SecretSanta-Excel-Helper
